fuse_core/test/launch_tests/test_parameters.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import unittest
+
+from ament_index_python.packages import get_package_prefix
+
+from launch import LaunchContext, LaunchDescription, LaunchService
+from launch.actions import EmitEvent, ExecuteProcess
+from launch.events import Shutdown
+
+import launch_pytest
+from launch_pytest.actions import ReadyToTest
+from launch_pytest.tools import process as process_tools
+
+from launch_ros.substitutions import FindPackageShare
+
+import pytest
+
+@pytest.fixture
+def test_proc():
+    test_root = os.path.join(
+        get_package_prefix('fuse_core'), '..', '..', 'build', 'fuse_core', 'test'
+    )
+
+    param_path = os.path.join(test_root, 'launch_tests', 'test_parameters.yaml')
+    test_path = os.path.join(test_root, 'test_parameters')
+
+    cmd = [test_path, '--ros-args', '--params-file', param_path]
+
+    return ExecuteProcess(cmd=cmd, shell=True, output='both', cached_output=True)
+
+@launch_pytest.fixture
+def generate_test_description(test_proc):
+    return LaunchDescription([test_proc, ReadyToTest()])
+
+@pytest.mark.launch(fixture=generate_test_description)
+def test_no_failed_gtests(test_proc, launch_context):
+    def validate_output(output):
+        assert '[  PASSED  ] 2 tests.' in output.splitlines(), (
+            'process never printed required passes')
```

Comment:
It looks like the test is returning `RUN_ALL_TESTS();`. I'd bet the exit code would be non-zero on failure, and so that could be the check here.